src/rgw/rgw_service.cc

```diff
@@ -199,12 +206,6 @@ int RGWServices_Def::init(CephContext *cct,
   }
 
   if (!raw) {
-    r = datalog_rados->start();
```

Comment:
why do we need to initialize the datalog for 'raw' storage when we didn't before?

Reply:
It's still initialized under `!raw`, see line 143. Just after 'zone' is initialized since FIFO needs to be able to get the pool from zone info.